### ISSUE TITLE
Option for Provisioned Concurrency in Server-Less SageMaker

### ISSUE THREAD

**Issue by jkubajek** [hashicorp/terraform#33215]
### Terraform Version

```shell
4.67.0
```

### Use Cases

I would like to set-up the SageMaker server-less inference instance and I would like to specify the `ProvisionedConcurrency` attribute. Here you can find an official blog post from AWS on how to set-up `ProvisionedConcurrency` manually - https://aws.amazon.com/blogs/machine-learning/announcing-provisioned-concurrency-for-amazon-sagemaker-serverless-inference/

### Attempted Solutions

Currently, Terraform in the `serverless_config` allows to specify only `max_concurrency` and `memory_size_in_mb`. https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_endpoint_configuration#serverless_config

### Proposal

I would like to have additional optional field in the `serverless_config` called `provisioned_concurrency` which would allow me to specify the `ProvisionedConcurrency` in the configuration of SageMaker.

**Comment by apparentlymart** [comment]
Hi @vivekprm,

I'm not familiar with what you're describing but based on your link to the `hashicorp/aws` provider docs I'm assuming that this is AWS provider feedback rather than Terraform feedback.

There's no AWS provider code in this repository, so for anything related to that provider you'll need to open an issue in [the provider's own repository](https://github.com/hashicorp/terraform-provider-aws).

I'm going to close this issue only because there's nothing we would change in _this_ repository to help achieve the result you are hoping for. Thanks!

**Comment by vivekprm** [comment]
Yeah Got it..I saw that terraform have different provider repos and this ticket should have been in aws provider repo.
